apps/gallery/test/marionette/edit_image_test.js

```diff
@@ -33,34 +33,38 @@ marionette('editing an image', function() {
   });
 
   test('should have crop options', function() {
-    app.enterMainEditScreen();
```

Comment:
I removed this method, and replaced with tap thumbnail and editbutton press to make the action more transparent